### ISSUE TITLE
Focusable close button

### ISSUE THREAD

**Issue by jasonbahl** [wp-graphql/wpgraphql-ide#9]
When opening the GraphiQL IDE in the drawer, there should be a focusable button to close the drawer. Currently you can click outside of the drawer to close it, but a focusable button is needed for accessibility.
